[2025-05-19T20:37:57.073+0000] {processor.py:161} INFO - Started process (PID=40) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:37:57.085+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:37:57.101+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:37:57.100+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:38:01.230+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:38:01.223+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:38:01.232+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:38:01.294+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 4.285 seconds
[2025-05-19T20:38:31.511+0000] {processor.py:161} INFO - Started process (PID=101) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:38:31.514+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:38:31.517+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:38:31.517+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:38:32.498+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:38:32.489+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:38:32.501+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:38:32.530+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.025 seconds
[2025-05-19T20:39:03.089+0000] {processor.py:161} INFO - Started process (PID=159) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:39:03.091+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:39:03.097+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:39:03.096+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:39:04.194+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:39:04.187+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:39:04.197+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:39:04.232+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.150 seconds
[2025-05-19T20:39:34.698+0000] {processor.py:161} INFO - Started process (PID=219) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:39:34.700+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:39:34.709+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:39:34.709+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:39:35.817+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:39:35.804+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:39:35.821+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:39:35.866+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.176 seconds
[2025-05-19T20:40:06.317+0000] {processor.py:161} INFO - Started process (PID=281) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:40:06.319+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:40:06.323+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:40:06.322+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:40:07.539+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:40:07.533+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:40:07.541+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:40:07.571+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.260 seconds
[2025-05-19T20:40:38.338+0000] {processor.py:161} INFO - Started process (PID=338) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:40:38.341+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:40:38.348+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:40:38.347+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:40:39.496+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:40:39.484+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:40:39.500+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:40:39.541+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.211 seconds
[2025-05-19T20:41:09.889+0000] {processor.py:161} INFO - Started process (PID=399) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:41:09.892+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:41:09.898+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:41:09.897+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:41:10.764+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:41:10.758+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:41:10.766+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:41:10.794+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.916 seconds
[2025-05-19T20:41:41.056+0000] {processor.py:161} INFO - Started process (PID=456) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:41:41.059+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:41:41.065+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:41:41.064+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:41:42.097+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:41:42.085+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:41:42.102+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:41:42.169+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.124 seconds
[2025-05-19T20:42:12.437+0000] {processor.py:161} INFO - Started process (PID=516) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:42:12.439+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:42:12.443+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:42:12.442+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:42:13.464+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:42:13.457+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:42:13.466+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:42:13.498+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.068 seconds
[2025-05-19T20:42:43.685+0000] {processor.py:161} INFO - Started process (PID=576) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:42:43.688+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:42:43.692+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:42:43.691+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:42:44.485+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:42:44.478+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:42:44.487+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:42:44.522+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.845 seconds
[2025-05-19T20:43:15.536+0000] {processor.py:161} INFO - Started process (PID=636) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:43:15.540+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:43:15.547+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:43:15.546+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:43:16.625+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:43:16.619+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:43:16.627+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:43:16.657+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.133 seconds
[2025-05-19T20:43:46.827+0000] {processor.py:161} INFO - Started process (PID=696) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:43:46.829+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:43:46.832+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:43:46.832+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:43:47.543+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:43:47.536+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:43:47.546+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:43:47.573+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.753 seconds
[2025-05-19T20:44:17.744+0000] {processor.py:161} INFO - Started process (PID=756) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:44:17.746+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:44:17.750+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:44:17.749+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:44:18.574+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:44:18.568+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:44:18.577+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:44:18.609+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.872 seconds
[2025-05-19T20:44:48.730+0000] {processor.py:161} INFO - Started process (PID=815) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:44:48.733+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:44:48.738+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:44:48.737+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:44:49.816+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:44:49.807+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:44:49.819+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:44:49.857+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.134 seconds
[2025-05-19T20:45:20.004+0000] {processor.py:161} INFO - Started process (PID=875) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:45:20.006+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:45:20.010+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:45:20.010+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:45:20.889+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:45:20.881+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:45:20.892+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:45:20.920+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.924 seconds
[2025-05-19T20:45:51.771+0000] {processor.py:161} INFO - Started process (PID=932) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:45:51.774+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:45:51.778+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:45:51.778+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:45:52.788+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:45:52.783+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:45:52.790+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:45:52.822+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.059 seconds
[2025-05-19T20:46:22.926+0000] {processor.py:161} INFO - Started process (PID=989) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:46:22.928+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:46:22.934+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:46:22.933+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:46:23.930+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:46:23.916+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:46:23.935+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:46:23.971+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.051 seconds
[2025-05-19T20:46:54.109+0000] {processor.py:161} INFO - Started process (PID=1052) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:46:54.111+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:46:54.116+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:46:54.115+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:46:55.478+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:46:55.462+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:46:55.484+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:46:55.526+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.423 seconds
[2025-05-19T20:47:25.901+0000] {processor.py:161} INFO - Started process (PID=1113) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:47:25.903+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:47:25.906+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:47:25.905+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:47:26.675+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:47:26.668+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:47:26.678+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:47:26.710+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.815 seconds
[2025-05-19T20:47:57.653+0000] {processor.py:161} INFO - Started process (PID=1170) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:47:57.654+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:47:57.658+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:47:57.657+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:47:58.440+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:47:58.433+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:47:58.442+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:47:58.471+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.825 seconds
[2025-05-19T20:48:29.154+0000] {processor.py:161} INFO - Started process (PID=1231) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:48:29.159+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:48:29.171+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:48:29.168+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:48:30.119+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:48:30.113+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:48:30.121+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:48:30.151+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.008 seconds
[2025-05-19T20:49:00.232+0000] {processor.py:161} INFO - Started process (PID=1294) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:49:00.234+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:49:00.239+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:49:00.238+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:49:01.157+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:49:01.152+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:49:01.160+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:49:01.193+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.966 seconds
[2025-05-19T20:49:31.949+0000] {processor.py:161} INFO - Started process (PID=1347) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:49:31.953+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:49:31.959+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:49:31.958+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:49:33.143+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:49:33.134+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:49:33.146+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:49:33.185+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.248 seconds
[2025-05-19T20:50:04.098+0000] {processor.py:161} INFO - Started process (PID=1408) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:50:04.100+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:50:04.102+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:50:04.102+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:50:04.981+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:50:04.976+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:50:04.983+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:50:05.010+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.917 seconds
[2025-05-19T20:50:35.666+0000] {processor.py:161} INFO - Started process (PID=1466) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:50:35.670+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:50:35.677+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:50:35.676+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:50:36.605+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:50:36.598+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:50:36.607+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:50:36.636+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.984 seconds
[2025-05-19T20:51:06.938+0000] {processor.py:161} INFO - Started process (PID=1523) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:51:06.939+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:51:06.942+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:51:06.942+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:51:07.676+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:51:07.670+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:51:07.679+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:51:07.709+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.776 seconds
[2025-05-19T20:51:38.376+0000] {processor.py:161} INFO - Started process (PID=1580) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:51:38.380+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:51:38.388+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:51:38.387+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:51:39.837+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:51:39.821+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:51:39.842+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:51:39.903+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.539 seconds
[2025-05-19T20:52:10.259+0000] {processor.py:161} INFO - Started process (PID=1637) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:52:10.261+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:52:10.265+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:52:10.265+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:52:11.215+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:52:11.209+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:52:11.217+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:52:11.248+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.999 seconds
[2025-05-19T20:52:41.510+0000] {processor.py:161} INFO - Started process (PID=1695) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:52:41.518+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:52:41.524+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:52:41.523+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:52:42.589+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:52:42.582+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:52:42.592+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:52:42.623+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.136 seconds
[2025-05-19T20:53:12.783+0000] {processor.py:161} INFO - Started process (PID=1752) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:53:12.786+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:53:12.790+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:53:12.789+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:53:13.747+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:53:13.737+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:53:13.751+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:53:13.788+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.012 seconds
[2025-05-19T20:53:44.499+0000] {processor.py:161} INFO - Started process (PID=1809) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:53:44.503+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:53:44.513+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:53:44.511+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:53:45.585+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:53:45.578+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:53:45.588+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:53:45.618+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.132 seconds
[2025-05-19T20:54:16.217+0000] {processor.py:161} INFO - Started process (PID=1867) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:54:16.219+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:54:16.222+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:54:16.221+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:54:17.180+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:54:17.173+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:54:17.183+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:54:17.218+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.008 seconds
[2025-05-19T20:57:49.497+0000] {processor.py:161} INFO - Started process (PID=40) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:57:49.503+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:57:49.514+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:57:49.512+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:57:53.284+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:57:53.270+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:57:53.302+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:57:53.351+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 3.883 seconds
[2025-05-19T20:58:23.545+0000] {processor.py:161} INFO - Started process (PID=100) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:58:23.547+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:58:23.551+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:58:23.550+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:58:24.641+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:58:24.626+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:58:24.646+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:58:24.703+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.169 seconds
[2025-05-19T20:58:55.039+0000] {processor.py:161} INFO - Started process (PID=161) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:58:55.047+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:58:55.054+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:58:55.053+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:58:55.996+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:58:55.991+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:58:55.998+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:58:56.021+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.001 seconds
[2025-05-19T20:59:26.921+0000] {processor.py:161} INFO - Started process (PID=221) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:59:26.924+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:59:26.930+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:59:26.930+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:59:27.789+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:59:27.780+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:59:27.792+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:59:27.828+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.918 seconds
[2025-05-19T20:59:57.914+0000] {processor.py:161} INFO - Started process (PID=279) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:59:57.916+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T20:59:57.920+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:59:57.919+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:59:58.646+0000] {logging_mixin.py:188} INFO - [2025-05-19T20:59:58.641+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T20:59:58.648+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T20:59:58.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.771 seconds
[2025-05-19T21:00:29.014+0000] {processor.py:161} INFO - Started process (PID=336) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:00:29.019+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:00:29.029+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:00:29.027+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:00:30.114+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:00:30.108+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:00:30.116+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:00:30.147+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.149 seconds
[2025-05-19T21:01:00.358+0000] {processor.py:161} INFO - Started process (PID=395) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:01:00.360+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:01:00.363+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:01:00.363+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:01:01.264+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:01:01.254+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:01:01.268+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:01:01.305+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.953 seconds
[2025-05-19T21:01:31.653+0000] {processor.py:161} INFO - Started process (PID=453) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:01:31.655+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:01:31.659+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:01:31.658+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:01:32.731+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:01:32.723+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:01:32.734+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:01:32.761+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.115 seconds
[2025-05-19T21:02:02.974+0000] {processor.py:161} INFO - Started process (PID=514) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:02:02.976+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:02:02.980+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:02:02.979+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:02:03.959+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:02:03.949+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:02:03.962+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:02:04.020+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.052 seconds
[2025-05-19T21:02:34.531+0000] {processor.py:161} INFO - Started process (PID=573) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:02:34.534+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:02:34.538+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:02:34.538+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:02:35.958+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:02:35.951+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:02:35.960+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:02:35.997+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.472 seconds
[2025-05-19T21:03:06.112+0000] {processor.py:161} INFO - Started process (PID=634) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:03:06.113+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:03:06.119+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:03:06.119+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:03:06.976+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:03:06.969+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:03:06.978+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:03:07.009+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.903 seconds
[2025-05-19T21:03:37.618+0000] {processor.py:161} INFO - Started process (PID=692) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:03:37.621+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:03:37.625+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:03:37.624+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:03:38.396+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:03:38.388+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:03:38.399+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:03:38.438+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.827 seconds
[2025-05-19T21:04:08.524+0000] {processor.py:161} INFO - Started process (PID=753) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:04:08.526+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:04:08.529+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:04:08.529+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:04:09.690+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:04:09.680+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:04:09.692+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:04:09.722+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.203 seconds
[2025-05-19T21:04:40.096+0000] {processor.py:161} INFO - Started process (PID=810) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:04:40.099+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:04:40.104+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:04:40.103+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:04:41.287+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:04:41.280+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:04:41.290+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:04:41.323+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.236 seconds
[2025-05-19T21:05:11.617+0000] {processor.py:161} INFO - Started process (PID=870) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:05:11.621+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:05:11.627+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:05:11.626+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:05:12.698+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:05:12.681+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:05:12.701+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:05:12.736+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.126 seconds
[2025-05-19T21:05:43.246+0000] {processor.py:161} INFO - Started process (PID=930) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:05:43.251+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:05:43.258+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:05:43.257+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:05:44.346+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:05:44.338+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:05:44.349+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:05:44.381+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.150 seconds
[2025-05-19T21:06:14.590+0000] {processor.py:161} INFO - Started process (PID=990) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:06:14.592+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:06:14.596+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:06:14.595+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:06:15.559+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:06:15.553+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:06:15.563+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:06:15.604+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.020 seconds
[2025-05-19T21:06:45.870+0000] {processor.py:161} INFO - Started process (PID=1051) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:06:45.877+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:06:45.882+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:06:45.881+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:06:46.929+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:06:46.919+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:06:46.932+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:06:46.973+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 1.113 seconds
[2025-05-19T21:07:17.180+0000] {processor.py:161} INFO - Started process (PID=1109) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:07:17.181+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:07:17.186+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:07:17.185+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:07:18.022+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:07:18.015+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:07:18.024+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:07:18.064+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.890 seconds
[2025-05-19T21:07:48.208+0000] {processor.py:161} INFO - Started process (PID=1170) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:07:48.209+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:07:48.211+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:07:48.211+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:07:48.827+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:07:48.822+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:07:48.828+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:07:48.845+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.643 seconds
[2025-05-19T21:08:18.916+0000] {processor.py:161} INFO - Started process (PID=1228) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:08:18.917+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:08:18.919+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:08:18.919+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:08:19.314+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:08:19.310+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:08:19.315+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:08:19.333+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.420 seconds
[2025-05-19T21:08:50.111+0000] {processor.py:161} INFO - Started process (PID=1285) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:08:50.112+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:08:50.113+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:08:50.113+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:08:50.526+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:08:50.523+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:08:50.527+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:08:50.544+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.436 seconds
[2025-05-19T21:09:21.450+0000] {processor.py:161} INFO - Started process (PID=1348) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:09:21.451+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:09:21.453+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:09:21.453+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:09:21.867+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:09:21.864+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:09:21.868+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:09:21.883+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.436 seconds
[2025-05-19T21:09:52.591+0000] {processor.py:161} INFO - Started process (PID=1405) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:09:52.592+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:09:52.595+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:09:52.594+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:09:53.007+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:09:53.005+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:09:53.008+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:09:53.027+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.438 seconds
[2025-05-19T21:10:24.026+0000] {processor.py:161} INFO - Started process (PID=1462) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:10:24.027+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:10:24.029+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:10:24.028+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:10:24.433+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:10:24.430+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:10:24.434+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:10:24.453+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.429 seconds
[2025-05-19T21:10:55.212+0000] {processor.py:161} INFO - Started process (PID=1519) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:10:55.213+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:10:55.215+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:10:55.215+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:10:55.618+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:10:55.615+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:10:55.619+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:10:55.636+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.427 seconds
[2025-05-19T21:11:26.568+0000] {processor.py:161} INFO - Started process (PID=1576) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:11:26.574+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:11:26.577+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:11:26.576+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:11:27.085+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:11:27.082+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:11:27.086+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:11:27.103+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.538 seconds
[2025-05-19T21:11:57.823+0000] {processor.py:161} INFO - Started process (PID=1633) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:11:57.824+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:11:57.826+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:11:57.826+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:11:58.241+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:11:58.239+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:11:58.242+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:11:58.262+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.442 seconds
[2025-05-19T21:12:29.154+0000] {processor.py:161} INFO - Started process (PID=1690) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:12:29.155+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:12:29.158+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:12:29.158+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:12:29.558+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:12:29.555+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:12:29.559+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:12:29.572+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.423 seconds
[2025-05-19T21:12:59.634+0000] {processor.py:161} INFO - Started process (PID=1747) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:12:59.635+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:12:59.637+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:12:59.637+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:13:00.046+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:13:00.044+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:13:00.047+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:13:00.064+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.433 seconds
[2025-05-19T21:13:31.060+0000] {processor.py:161} INFO - Started process (PID=1804) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:13:31.061+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:13:31.063+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:13:31.063+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:13:31.465+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:13:31.461+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:13:31.465+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:13:31.483+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.427 seconds
[2025-05-19T21:14:02.425+0000] {processor.py:161} INFO - Started process (PID=1861) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:14:02.427+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:14:02.431+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:14:02.430+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:14:03.029+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:14:03.026+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:14:03.030+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:14:03.051+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.630 seconds
[2025-05-19T21:14:33.909+0000] {processor.py:161} INFO - Started process (PID=1918) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:14:33.910+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:14:33.911+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:14:33.911+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:14:34.322+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:14:34.319+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:14:34.322+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:14:34.341+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.434 seconds
[2025-05-19T21:15:05.285+0000] {processor.py:161} INFO - Started process (PID=1975) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:15:05.286+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:15:05.288+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:15:05.288+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:15:05.779+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:15:05.774+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:15:05.780+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:15:05.795+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.513 seconds
[2025-05-19T21:15:36.735+0000] {processor.py:161} INFO - Started process (PID=2032) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:15:36.736+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:15:36.737+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:15:36.737+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:15:37.162+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:15:37.160+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:15:37.163+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:15:37.181+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.449 seconds
[2025-05-19T21:16:07.600+0000] {processor.py:161} INFO - Started process (PID=2089) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:16:07.601+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:16:07.603+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:16:07.603+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:16:07.993+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:16:07.990+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:16:07.994+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:16:08.011+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.413 seconds
[2025-05-19T21:16:38.924+0000] {processor.py:161} INFO - Started process (PID=2146) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:16:38.925+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:16:38.926+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:16:38.926+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:16:39.311+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:16:39.309+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:16:39.312+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:16:39.325+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.406 seconds
[2025-05-19T21:17:09.987+0000] {processor.py:161} INFO - Started process (PID=2203) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:17:09.988+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:17:09.989+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:17:09.989+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:17:10.374+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:17:10.372+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:17:10.375+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:17:10.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.416 seconds
[2025-05-19T21:17:40.724+0000] {processor.py:161} INFO - Started process (PID=2260) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:17:40.725+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:17:40.728+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:17:40.727+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:17:41.181+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:17:41.178+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:17:41.182+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:17:41.201+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.484 seconds
[2025-05-19T21:18:11.367+0000] {processor.py:161} INFO - Started process (PID=2321) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:18:11.368+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:18:11.370+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:11.370+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:18:11.810+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:11.807+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/flight_price_pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/flight_price_pipeline.py", line 9, in <module>
    from tasks.utils.logging_config import configure_logging
ImportError: cannot import name 'configure_logging' from 'tasks.utils.logging_config' (/opt/airflow/dags/tasks/utils/logging_config.py)
[2025-05-19T21:18:11.810+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:18:11.830+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.465 seconds
[2025-05-19T21:18:42.048+0000] {processor.py:161} INFO - Started process (PID=2378) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:18:42.049+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:18:42.051+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:42.051+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:18:42.582+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:18:42.696+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:42.695+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:flight_price_analysis
[2025-05-19T21:18:42.697+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:18:42,695 - airflow.auth.managers.fab.security_manager.override - INFO - Created Permission View: can read on DAG:flight_price_analysis
[2025-05-19T21:18:42.709+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:42.708+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:flight_price_analysis
[2025-05-19T21:18:42.709+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:18:42,708 - airflow.auth.managers.fab.security_manager.override - INFO - Created Permission View: can delete on DAG:flight_price_analysis
[2025-05-19T21:18:42.721+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:42.721+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:flight_price_analysis
[2025-05-19T21:18:42.722+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:18:42,721 - airflow.auth.managers.fab.security_manager.override - INFO - Created Permission View: can edit on DAG:flight_price_analysis
[2025-05-19T21:18:42.723+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:42.722+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:18:42.723+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:18:42,722 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:18:42.735+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:42.734+0000] {dag.py:3058} INFO - Creating ORM DAG for flight_price_analysis
[2025-05-19T21:18:42.736+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:18:42,734 - airflow.models.dag - INFO - Creating ORM DAG for flight_price_analysis
[2025-05-19T21:18:42.738+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:18:42.737+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:18:42.741+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:18:42,737 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:18:42.776+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.730 seconds
[2025-05-19T21:19:13.040+0000] {processor.py:161} INFO - Started process (PID=2435) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:19:13.041+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:19:13.044+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:19:13.044+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:19:13.494+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:19:13.510+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:19:13.510+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:19:13.511+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:19:13,510 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:19:13.522+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:19:13.521+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:19:13.523+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:19:13,521 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:19:13.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.506 seconds
[2025-05-19T21:19:43.597+0000] {processor.py:161} INFO - Started process (PID=2493) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:19:43.598+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:19:43.600+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:19:43.600+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:19:44.019+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:19:44.032+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:19:44.032+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:19:44.033+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:19:44,032 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:19:44.044+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:19:44.044+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:19:44.045+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:19:44,044 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:19:44.069+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.474 seconds
[2025-05-19T21:20:14.348+0000] {processor.py:161} INFO - Started process (PID=2545) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:20:14.349+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:20:14.350+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:20:14.350+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:20:14.803+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:20:14.818+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:20:14.818+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:20:14.819+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:20:14,818 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:20:14.830+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:20:14.830+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:20:14.831+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:20:14,830 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:20:14.864+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.519 seconds
[2025-05-19T21:20:45.346+0000] {processor.py:161} INFO - Started process (PID=2602) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:20:45.347+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:20:45.348+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:20:45.348+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:20:45.816+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:20:45.830+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:20:45.830+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:20:45.831+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:20:45,830 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:20:45.843+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:20:45.843+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:20:45.844+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:20:45,843 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:20:45.869+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.526 seconds
[2025-05-19T21:21:16.485+0000] {processor.py:161} INFO - Started process (PID=2660) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:21:16.487+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:21:16.489+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:21:16.489+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:21:17.114+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:21:17.150+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:21:17.149+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:21:17.154+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:21:17,149 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:21:17.181+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:21:17.180+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:21:17.181+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:21:17,180 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:21:17.207+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.725 seconds
[2025-05-19T21:21:48.099+0000] {processor.py:161} INFO - Started process (PID=2721) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:21:48.100+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:21:48.102+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:21:48.101+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:21:48.699+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:21:48.722+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:21:48.722+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:21:48.723+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:21:48,722 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:21:48.738+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:21:48.737+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:21:48.738+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:21:48,737 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:21:48.764+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.668 seconds
[2025-05-19T21:22:18.931+0000] {processor.py:161} INFO - Started process (PID=2782) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:22:18.932+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:22:18.934+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:22:18.933+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:22:19.385+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:22:19.399+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:22:19.398+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:22:19.400+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:22:19,398 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:22:19.414+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:22:19.413+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:22:19.415+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:22:19,413 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:22:19.447+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.519 seconds
[2025-05-19T21:22:50.143+0000] {processor.py:161} INFO - Started process (PID=2839) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:22:50.144+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:22:50.146+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:22:50.145+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:22:50.422+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:22:50.438+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:22:50.437+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:22:50.439+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:22:50,437 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:22:50.449+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:22:50.449+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:22:50.450+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:22:50,449 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:22:50.472+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.333 seconds
[2025-05-19T21:23:00.973+0000] {processor.py:161} INFO - Started process (PID=2881) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:23:00.974+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:23:00.976+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:23:00.975+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:23:01.342+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:23:01.452+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:23:01.451+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:23:01.453+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:23:01,451 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:23:01.465+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:23:01.465+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:23:01.466+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:23:01,465 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:23:01.497+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.526 seconds
[2025-05-19T21:23:32.326+0000] {processor.py:161} INFO - Started process (PID=2942) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:23:32.327+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:23:32.329+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:23:32.328+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:23:32.616+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:23:32.633+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:23:32.633+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:23:32.634+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:23:32,633 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:23:32.647+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:23:32.646+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:23:32.648+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:23:32,646 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:23:32.668+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.346 seconds
[2025-05-19T21:24:02.710+0000] {processor.py:161} INFO - Started process (PID=3002) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:24:02.711+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:24:02.713+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:24:02.713+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:24:03.029+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:24:03.049+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:24:03.048+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:24:03.050+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:24:03,048 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:24:03.066+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:24:03.066+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:24:03.067+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:24:03,066 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:24:03.116+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.409 seconds
[2025-05-19T21:24:33.295+0000] {processor.py:161} INFO - Started process (PID=3060) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:24:33.297+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:24:33.299+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:24:33.299+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:24:33.616+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:24:33.636+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:24:33.635+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:24:33.637+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:24:33,635 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:24:33.649+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:24:33.648+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:24:33.650+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:24:33,648 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:24:33.676+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.385 seconds
[2025-05-19T21:25:04.489+0000] {processor.py:161} INFO - Started process (PID=3120) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:25:04.490+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:25:04.493+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:25:04.492+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:25:04.894+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:25:04.912+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:25:04.912+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:25:04.913+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:25:04,912 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:25:04.925+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:25:04.924+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:25:04.925+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:25:04,924 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:25:04.945+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.460 seconds
[2025-05-19T21:25:35.046+0000] {processor.py:161} INFO - Started process (PID=3183) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:25:35.048+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:25:35.051+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:25:35.050+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:25:35.432+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:25:35.455+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:25:35.455+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:25:35.456+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:25:35,455 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:25:35.470+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:25:35.470+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:25:35.471+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:25:35,470 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:25:35.501+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.457 seconds
[2025-05-19T21:26:06.148+0000] {processor.py:161} INFO - Started process (PID=3242) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:06.149+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:26:06.151+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:06.151+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:06.473+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:06.491+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:06.490+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:26:06.492+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:26:06,490 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:26:06.504+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:06.504+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:26:06.505+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:26:06,504 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:26:06.533+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.387 seconds
[2025-05-19T21:26:37.459+0000] {processor.py:161} INFO - Started process (PID=3301) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:37.460+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:26:37.462+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:37.461+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:37.767+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:37.783+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:37.783+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:26:37.784+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:26:37,783 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:26:37.796+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:37.796+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:26:37.796+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:26:37,796 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:26:37.815+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.358 seconds
[2025-05-19T21:26:48.031+0000] {processor.py:161} INFO - Started process (PID=3304) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:48.032+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:26:48.033+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:48.033+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:48.335+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:26:48.422+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:48.421+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:26:48.423+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:26:48,421 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:26:48.432+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:26:48.432+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:26:48.433+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:26:48,432 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:26:48.476+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.448 seconds
[2025-05-19T21:27:19.387+0000] {processor.py:161} INFO - Started process (PID=3363) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:27:19.388+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:27:19.390+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:27:19.389+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:27:19.688+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:27:19.704+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:27:19.704+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:27:19.705+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:27:19,704 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:27:19.717+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:27:19.717+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:27:19.718+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:27:19,717 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:27:19.737+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.353 seconds
[2025-05-19T21:27:50.501+0000] {processor.py:161} INFO - Started process (PID=3422) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:27:50.502+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:27:50.504+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:27:50.504+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:27:50.843+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:27:50.862+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:27:50.862+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:27:50.863+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:27:50,862 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:27:50.875+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:27:50.875+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:27:50.876+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:27:50,875 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:27:50.898+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.401 seconds
[2025-05-19T21:28:21.479+0000] {processor.py:161} INFO - Started process (PID=3481) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:28:21.480+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:28:21.482+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:28:21.482+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:28:21.843+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:28:21.873+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:28:21.872+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:28:21.874+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:28:21,872 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:28:21.890+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:28:21.890+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:28:21.891+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:28:21,890 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:28:21.912+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.436 seconds
[2025-05-19T21:28:52.141+0000] {processor.py:161} INFO - Started process (PID=3547) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:28:52.142+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:28:52.144+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:28:52.144+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:28:52.445+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:28:52.462+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:28:52.462+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:28:52.463+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:28:52,462 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:28:52.475+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:28:52.475+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:28:52.476+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:28:52,475 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:28:52.497+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.359 seconds
[2025-05-19T21:29:22.940+0000] {processor.py:161} INFO - Started process (PID=3606) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:22.942+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:29:22.944+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:22.943+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:23.243+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:23.263+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:23.262+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:29:23.264+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:29:23,262 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:29:23.279+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:23.278+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:29:23.280+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:29:23,278 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:29:23.315+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.378 seconds
[2025-05-19T21:29:27.829+0000] {processor.py:161} INFO - Started process (PID=3618) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:27.830+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:29:27.833+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:27.833+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:28.141+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:28.224+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:28.224+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:29:28.225+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:29:28,224 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:29:28.237+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:28.236+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:29:28.238+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:29:28,236 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:29:28.263+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.437 seconds
[2025-05-19T21:29:58.684+0000] {processor.py:161} INFO - Started process (PID=3677) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:58.686+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:29:58.688+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:58.687+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:58.983+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:29:59.001+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:59.000+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:29:59.001+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:29:59,000 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:29:59.013+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:29:59.012+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:29:59.014+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:29:59,012 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:29:59.039+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.358 seconds
[2025-05-19T21:30:29.096+0000] {processor.py:161} INFO - Started process (PID=3736) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:30:29.097+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:30:29.100+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:30:29.099+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:30:29.435+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:30:29.450+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:30:29.450+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:30:29.451+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:30:29,450 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:30:29.462+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:30:29.462+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:30:29.463+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:30:29,462 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:30:29.487+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.395 seconds
[2025-05-19T21:30:59.549+0000] {processor.py:161} INFO - Started process (PID=3798) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:30:59.550+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:30:59.552+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:30:59.552+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:30:59.881+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:30:59.900+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:30:59.899+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:30:59.901+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:30:59,899 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:30:59.914+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:30:59.914+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:30:59.915+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:30:59,914 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:30:59.943+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.398 seconds
[2025-05-19T21:31:30.099+0000] {processor.py:161} INFO - Started process (PID=3854) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:31:30.101+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:31:30.103+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:31:30.103+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:31:30.499+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:31:30.518+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:31:30.517+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:31:30.518+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:31:30,517 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:31:30.533+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:31:30.533+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:31:30.534+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:31:30,533 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:31:30.562+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.466 seconds
[2025-05-19T21:32:00.681+0000] {processor.py:161} INFO - Started process (PID=3916) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:32:00.682+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:32:00.684+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:32:00.683+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:32:00.978+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:32:00.997+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:32:00.996+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:32:00.998+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:32:00,996 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:32:01.010+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:32:01.010+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:32:01.011+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:32:01,010 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:32:01.037+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.358 seconds
[2025-05-19T21:32:31.395+0000] {processor.py:161} INFO - Started process (PID=3975) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:32:31.396+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:32:31.399+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:32:31.398+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:32:31.805+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:32:31.823+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:32:31.822+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:32:31.823+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:32:31,822 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:32:31.838+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:32:31.838+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:32:31.839+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:32:31,838 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:32:31.869+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.479 seconds
[2025-05-19T21:33:01.958+0000] {processor.py:161} INFO - Started process (PID=4034) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:33:01.959+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:33:01.961+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:33:01.961+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:33:02.365+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:33:02.382+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:33:02.381+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:33:02.382+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:33:02,381 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:33:02.393+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:33:02.393+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:33:02.394+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:33:02,393 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:33:02.410+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.456 seconds
[2025-05-19T21:33:32.545+0000] {processor.py:161} INFO - Started process (PID=4095) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:33:32.545+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:33:32.547+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:33:32.547+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:33:32.846+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:33:32.862+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:33:32.862+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:33:32.863+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:33:32,862 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:33:32.874+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:33:32.874+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:33:32.875+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:33:32,874 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:33:32.894+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.353 seconds
[2025-05-19T21:34:03.006+0000] {processor.py:161} INFO - Started process (PID=4149) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:34:03.008+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:34:03.010+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:34:03.009+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:34:03.322+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:34:03.340+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:34:03.340+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:34:03.341+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:34:03,340 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:34:03.353+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:34:03.353+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:34:03.355+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:34:03,353 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:34:03.380+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.377 seconds
[2025-05-19T21:34:33.842+0000] {processor.py:161} INFO - Started process (PID=4208) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:34:33.844+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:34:33.846+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:34:33.846+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:34:34.172+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:34:34.193+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:34:34.193+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:34:34.194+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:34:34,193 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:34:34.211+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:34:34.211+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:34:34.212+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:34:34,211 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:34:34.243+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.405 seconds
[2025-05-19T21:35:04.390+0000] {processor.py:161} INFO - Started process (PID=4267) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:35:04.391+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:35:04.393+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:35:04.392+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:35:04.730+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:35:04.750+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:35:04.749+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:35:04.751+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:35:04,749 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:35:04.765+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:35:04.765+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:35:04.766+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:35:04,765 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:35:04.824+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.438 seconds
[2025-05-19T21:35:34.892+0000] {processor.py:161} INFO - Started process (PID=4329) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:35:34.893+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:35:34.894+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:35:34.894+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:35:35.194+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:35:35.211+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:35:35.211+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:35:35.212+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:35:35,211 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:35:35.223+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:35:35.223+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:35:35.224+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:35:35,223 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:35:35.244+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.355 seconds
[2025-05-19T21:36:05.356+0000] {processor.py:161} INFO - Started process (PID=4390) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:36:05.357+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:36:05.358+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:36:05.358+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:36:05.641+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:36:05.657+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:36:05.656+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:36:05.658+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:36:05,656 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:36:05.680+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:36:05.680+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:36:05.681+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:36:05,680 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:36:05.709+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.356 seconds
[2025-05-19T21:36:35.817+0000] {processor.py:161} INFO - Started process (PID=4444) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:36:35.818+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:36:35.820+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:36:35.820+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:36:36.167+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:36:36.187+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:36:36.187+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:36:36.188+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:36:36,187 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:36:36.206+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:36:36.206+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:36:36.207+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:36:36,206 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:36:36.230+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.416 seconds
[2025-05-19T21:37:06.313+0000] {processor.py:161} INFO - Started process (PID=4503) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:37:06.316+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:37:06.319+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:37:06.319+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:37:06.793+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:37:06.812+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:37:06.812+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:37:06.813+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:37:06,812 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:37:06.826+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:37:06.826+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:37:06.827+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:37:06,826 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:37:06.846+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.537 seconds
[2025-05-19T21:37:36.991+0000] {processor.py:161} INFO - Started process (PID=4565) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:37:36.992+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:37:36.993+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:37:36.993+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:37:37.411+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:37:37.427+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:37:37.426+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:37:37.427+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:37:37,426 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:37:37.438+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:37:37.438+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:37:37.439+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:37:37,438 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:37:37.461+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.473 seconds
[2025-05-19T21:38:07.896+0000] {processor.py:161} INFO - Started process (PID=4627) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:38:07.897+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:38:07.899+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:38:07.899+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:38:08.246+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:38:08.265+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:38:08.264+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:38:08.266+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:38:08,264 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:38:08.276+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:38:08.276+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:38:08.277+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:38:08,276 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:38:08.301+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.408 seconds
[2025-05-19T21:38:38.459+0000] {processor.py:161} INFO - Started process (PID=4688) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:38:38.460+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:38:38.462+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:38:38.462+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:38:38.757+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:38:38.774+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:38:38.773+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:38:38.774+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:38:38,773 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:38:38.786+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:38:38.786+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:38:38.787+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:38:38,786 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:38:38.808+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.352 seconds
[2025-05-19T21:39:08.938+0000] {processor.py:161} INFO - Started process (PID=4748) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:39:08.938+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:39:08.940+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:39:08.940+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:39:09.239+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:39:09.255+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:39:09.255+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:39:09.256+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:39:09,255 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:39:09.268+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:39:09.268+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:39:09.269+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:39:09,268 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:39:09.289+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.355 seconds
[2025-05-19T21:39:39.394+0000] {processor.py:161} INFO - Started process (PID=4806) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:39:39.394+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:39:39.396+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:39:39.396+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:39:39.667+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:39:39.684+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:39:39.683+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:39:39.684+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:39:39,683 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:39:39.695+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:39:39.695+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:39:39.696+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:39:39,695 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:39:39.715+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.324 seconds
[2025-05-19T21:40:09.933+0000] {processor.py:161} INFO - Started process (PID=4865) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:40:09.934+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:40:09.935+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:40:09.935+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:40:10.292+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:40:10.308+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:40:10.307+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:40:10.308+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:40:10,307 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:40:10.321+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:40:10.321+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:40:10.322+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:40:10,321 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:40:10.343+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.413 seconds
[2025-05-19T21:40:42.405+0000] {processor.py:161} INFO - Started process (PID=4916) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:40:42.406+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:40:42.408+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:40:42.408+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:40:42.736+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:40:42.753+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:40:42.752+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:40:42.754+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:40:42,752 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:40:42.766+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:40:42.766+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:40:42.767+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:40:42,766 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:40:42.790+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.388 seconds
[2025-05-19T21:41:13.115+0000] {processor.py:161} INFO - Started process (PID=4976) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:41:13.116+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:41:13.117+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:41:13.117+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:41:13.497+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:41:13.514+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:41:13.513+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:41:13.515+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:41:13,513 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:41:13.528+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:41:13.528+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:41:13.529+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:41:13,528 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:41:13.547+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.436 seconds
[2025-05-19T21:41:43.703+0000] {processor.py:161} INFO - Started process (PID=5039) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:41:43.704+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:41:43.706+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:41:43.706+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:41:44.203+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:41:44.224+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:41:44.223+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:41:44.224+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:41:44,223 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:41:44.238+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:41:44.238+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:41:44.239+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:41:44,238 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:41:44.263+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.562 seconds
[2025-05-19T21:42:14.328+0000] {processor.py:161} INFO - Started process (PID=5094) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:42:14.329+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:42:14.331+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:42:14.331+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:42:14.700+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:42:14.720+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:42:14.719+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:42:14.721+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:42:14,719 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:42:14.734+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:42:14.734+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:42:14.735+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:42:14,734 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:42:14.758+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.433 seconds
[2025-05-19T21:42:45.038+0000] {processor.py:161} INFO - Started process (PID=5153) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:42:45.039+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:42:45.040+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:42:45.040+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:42:45.332+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:42:45.352+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:42:45.351+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:42:45.352+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:42:45,351 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:42:45.365+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:42:45.365+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:42:45.366+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:42:45,365 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:42:45.388+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.354 seconds
[2025-05-19T21:43:15.614+0000] {processor.py:161} INFO - Started process (PID=5212) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:43:15.615+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:43:15.616+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:43:15.616+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:43:15.910+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:43:15.925+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:43:15.925+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:43:15.926+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:43:15,925 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:43:15.937+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:43:15.936+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:43:15.937+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:43:15,936 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:43:15.958+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.346 seconds
[2025-05-19T21:43:46.025+0000] {processor.py:161} INFO - Started process (PID=5271) to work on /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:43:46.026+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/flight_price_pipeline.py for tasks to queue
[2025-05-19T21:43:46.028+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:43:46.027+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:43:46.355+0000] {processor.py:840} INFO - DAG(s) 'flight_price_analysis' retrieved from /opt/airflow/dags/flight_price_pipeline.py
[2025-05-19T21:43:46.372+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:43:46.372+0000] {dag.py:3036} INFO - Sync 1 DAGs
[2025-05-19T21:43:46.373+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:43:46,372 - airflow.models.dag - INFO - Sync 1 DAGs
[2025-05-19T21:43:46.385+0000] {logging_mixin.py:188} INFO - [2025-05-19T21:43:46.385+0000] {dag.py:3823} INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:43:46.386+0000] {logging_mixin.py:188} WARNING - 2025-05-19 21:43:46,385 - airflow.models.dag - INFO - Setting next_dagrun for flight_price_analysis to None, run_after=None
[2025-05-19T21:43:46.406+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/flight_price_pipeline.py took 0.384 seconds
